examples/v2/nodejs/dockerfiles/node/service.js

```diff
@@ -0,0 +1,58 @@
+var http = require('http');
+var url = require('url');
+var mysql = require('mysql');
+
+var connection = mysql.createConnection({
+    port     : process.env.SEVEN_SERVICE_MYSQL_PORT,
+    host     : process.env.SEVEN_SERVICE_PROXY_HOST,
+    user     : 'admin',
```

Comment:
Should user and password be environmental variables?

Reply:
Possibly. I'm inclined to think that because this is an example, the risk is lower than if it was a full application, but maybe we want to use sensible best practices regardless.